tests/acceptance/resolve-guid-test.ts

```diff
@@ -113,8 +113,9 @@ module('Acceptance | resolve-guid', hooks => {
             });
 
             test('Forks', async assert => {
-                server.create('root', { currentUser: null, activeFlags: [] });
-                const reg = server.create('registration');
+                const currentUser = server.create('user');
+                server.create('root', { currentUser, activeFlags: [] });
```

Comment:
Why `activeFlags: []`?

Generally, you shouldn't have to create the `root` explicitly anymore. Instead, `currentUser = server.create('user', 'loggedIn')`.

Reply:
To test routing when `ember_registries_detail_page` flag is not set, since the flag is automatically added at `root` creation.
https://github.com/CenterForOpenScience/ember-osf-web/blob/e1be8e2a5a57ad914a8b5e73dae62e22437bee35/mirage/factories/root.ts#L24-L30

Reply:
In the latest `develop`, you can create a root with the the `oldRegistrationDetail` trait
https://github.com/CenterForOpenScience/ember-osf-web/blob/a9756531e95ddffdede1e37c7717408da9eae426/mirage/factories/root.ts#L44-L48